src/util/account.js

```diff
@@ -1,28 +1,34 @@
-import { EthereumNetworkIds, NetworkTypes } from '../constants';
+import { SubstrateNetworkKeys, NetworkProtocols } from '../constants';
 
 export function accountId({
   address,
-  networkType = 'ethereum',
-  networkKey = EthereumNetworkIds.FRONTIER
+  protocol = NetworkProtocols.SUBSTRATE,
+  publicKey,
+  networkKey = SubstrateNetworkKeys.SUBSTRATE
 }) {
   if (typeof address !== 'string' || address.length === 0) {
-    throw new Error(`Couldn't create an accountId, missing address`);
+    throw new Error(`Couldn't create an accountId, address missing`);
+  }
+  if (protocol === NetworkProtocols.SUBSTRATE){
+    return `${protocol}:${publicKey}`;
```

Comment:
@Tbaut what do you think about we format this instead to return `${protocol}:${ss58Address}:${genesisHash}`?

Reply:
yup that's what needs to be done now that we have the `genesisHash`.